elekto/templates/layouts/app.html

```diff
@@ -49,6 +49,21 @@
                 document.getElementById('message').hidden = true
             }, 10000);
         }
+ document.addEventListener('DOMContentLoaded', function () {
+        // Fetch the version from the Flask API
+        fetch('https://api.github.com/repos/elekto-io/elekto/releases/latest')
```

Comment:
Am I reading this right?  It seems to be pulling the latest version number from GitHub, rather than the version the user is actually running.  Or did I miss something?

Reply:
my bad , i haven't thought through this

Reply:
should i run the bash script ' git describe --tags from the flask ' to get the current version?